.gitignore

```diff
@@ -31,3 +31,6 @@
 **/fulcrum.iml
 **/token
 **/target/*
+# Ignore contents of releases folder, except README
+**/releases/*
```

Comment:
"releases" folder cannot be added to gitignore, because without checking-in release folder contents, gitops will not work.